app/models/concerns/findings/csv.rb

```diff
@@ -58,6 +58,16 @@ def to_csv_a corporate
 
   private
 
+    def show_commitment_date_required_level_text
+      state = I18n.t('findings.state.being_implemented')
+
+      if (Finding.show_commitment_support?) && (%W(#{state}).include? full_state_text)
+        return true
+      end
+
+      false
```

Comment:
Enano, mansa rosca has hecho, esa lógica es el equivalente a esto:
```suggestion
      Finding.show_commitment_support? && being_implemented?
```